scriptFile needs to be in the shaded jar but README suggests otherwise

The README suggests that the scriptFile can be some file somewhere in the project (`src/packaging/someScript.extension`) but actually it currently has to be in the shaded jar (https://github.com/brianm/really-executable-jars-maven-plugin/blob/master/src/main/java/org/skife/waffles/ReallyExecutableJarMojo.java#L169-170).
Either the README should be updated or the plugin should use file resolution instead of the classloader.

With the latest release version (1.4.0), I encounter this issue too. I understand too that the readme said that the configured script file is a direct path.
If the file cannot be found in the constructed jar, a NullPointerException occurs [here](https://github.com/brianm/really-executable-jars-maven-plugin/blob/master/src/main/java/org/skife/waffles/ReallyExecutableJarMojo.java#L172) because of the null InputStream `scriptIn` variable.

Stack trace:
```java
Caused by: java.lang.NullPointerException
        at java.io.Reader.<init>(Reader.java:78)
        at java.io.InputStreamReader.<init>(InputStreamReader.java:72)
        at org.codehaus.plexus.util.IOUtil.copy(IOUtil.java:248)
        at org.codehaus.plexus.util.IOUtil.toString(IOUtil.java:307)
        at org.codehaus.plexus.util.IOUtil.toString(IOUtil.java:295)
        at org.skife.waffles.ReallyExecutableJarMojo.makeExecutable(ReallyExecutableJarMojo.java:172)
        at org.skife.waffles.ReallyExecutableJarMojo.execute(ReallyExecutableJarMojo.java:129)
```
If the script must be read in the constructed jar, a log and null check may be added.

--- a/src/main/java/org/skife/waffles/ReallyExecutableJarMojo.java
+++ b/src/main/java/org/skife/waffles/ReallyExecutableJarMojo.java
@@ -164,7 +164,7 @@ public class ReallyExecutableJarMojo extends AbstractMojo
             if (scriptFile == null) {
                 out.write(("#!/bin/sh\n\nexec java " + flags + " -jar \"$0\" \"$@\"\n\n").getBytes("ASCII"));
             }
-            else if (Files.exists(Paths.get((scriptFile)))) {
+            else if (Files.exists(Paths.get(scriptFile))) {
                 getLog().debug(String.format("Loading file[%s] from filesystem", scriptFile));
 
                 byte[] script = Files.readAllBytes(Paths.get(scriptFile));

--- a/src/main/java/org/skife/waffles/ReallyExecutableJarMojo.java
+++ b/src/main/java/org/skife/waffles/ReallyExecutableJarMojo.java
@@ -163,7 +163,15 @@ public class ReallyExecutableJarMojo extends AbstractMojo
 
             if (scriptFile == null) {
                 out.write(("#!/bin/sh\n\nexec java " + flags + " -jar \"$0\" \"$@\"\n\n").getBytes("ASCII"));
-            } else {
+            }
+            else if (Files.exists(Paths.get((scriptFile)))) {
+                getLog().debug(String.format("Loading file[%s] from filesystem", scriptFile));
+
+                byte[] script = Files.readAllBytes(Paths.get(scriptFile));
+                out.write(script);
+                out.write(new byte[]{'\n', '\n'});
+            }
+            else {
                 getLog().debug(String.format("Loading file[%s] from jar[%s]", scriptFile, original));
 
                 try (final URLClassLoader loader = new URLClassLoader(new URL[]{original.toUri().toURL()}, null);

